bitnami/wordpress/values.yaml

```diff
@@ -570,3 +570,23 @@ sidecars: {}
 ##    command: ['sh', '-c', 'copy themes and plugins from git and push to /bitnami/wordpress/wp-content. Should work with extraVolumeMounts and extraVolumes']
 ##
 initContainers: {}
+
```

Comment:
Add this same content to the `values-production.yaml` file

Reply:
Done